### PR TITLE
[Changelog] Adds SR-4206 and move SR-8974 to Swift 5.1

### PR DESCRIPTION
**Pull request by theblixguy**
Adds changelog entry for SR-4206 (PR #24484) and moves SR-8974 to be under Swift 5.1 (PR #24919).

### CODE REVIEW

**Comment by theblixguy** [comment]
cc @slavapestov

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,27 +26,24 @@ CHANGELOG
 Swift Next
 ----------
 
-* [SR-8974][]:
+* [SR-4206][]:
 
-  Duplicate tuple element labels are no longer allowed, because it leads
-  to incorrect behavior. For example:
+  Overriden methods are no longer allowed to have a different generic
```
Comment (slavapestov) [inline CHANGELOG.md]: Note that they can have a different generic signature as long as it has fewer requirements than the base method (once we fix the thunking for that anyway :) ). I would phrase this as "A method override is no longer allowed to have a generic signature with requirements not imposed by the base method. For example, ..."

Comment (theblixguy) [reply CHANGELOG.md]: Done :)

**Review by slavapestov — APPROVED** [review]

**Comment by slavapestov** [comment]
@swift-ci Please smoke test and merge

**Comment by slavapestov** [comment]
Thanks!

**Comment by theblixguy** [comment]
Hmm I don't think the tests ran, do we need to run a test for a changelog? :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -68,6 +65,28 @@ Swift Next
...
+  let dupLabels: (foo: Int, foo: Int) = (foo: 1, foo: 2)
+
+  enum Foo { case bar(x: Int, x: Int) }
+  let f: Foo = .bar(x: 0, x: 1)
+  ```
+
+  will now be diagnosed as an error. 
+
+  Note: You can still use duplicate labels when declaring functions and
+  subscripts, as long as the internal labels are different. For example:
```
Comment (xwu) [inline CHANGELOG.md]: _The Swift Programming Language_ calls these "local parameter names."

```suggestion
  subscripts, as long as the local parameter names are different. For example:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -68,6 +65,28 @@ Swift Next
...
+  ```
+  let dupLabels: (foo: Int, foo: Int) = (foo: 1, foo: 2)
+
+  enum Foo { case bar(x: Int, x: Int) }
+  let f: Foo = .bar(x: 0, x: 1)
+  ```
+
+  will now be diagnosed as an error. 
+
+  Note: You can still use duplicate labels when declaring functions and
```
Comment (xwu) [inline CHANGELOG.md]: _The Swift Programming Language_ calls these "argument labels."

```suggestion
  Note: You can still use duplicate argument labels when declaring functions and
```

**Comment by theblixguy** [comment]
Thanks @xwu I’ve applied your suggestions.

**Comment by slavapestov** [comment]
@swift-ci Please smoke test and merge

**Comment by theblixguy** [comment]
Maybe smoke-test-and-merge is broken as it keeps failing

**Comment by slavapestov** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**